crates/hir_def/src/attr.rs

```diff
@@ -652,14 +654,14 @@ pub enum AttrInput {
     /// `#[attr = "string"]`
     Literal(SmolStr),
     /// `#[attr(subtree)]`
-    TokenTree(Subtree),
+    TokenTree(tt::Subtree, mbe::TokenMap),
```

Comment:
Yeah, it seems like we need to add TokenMap here, this is a fundamentally missing info.